src/node_crypto.cc

```diff
@@ -1202,6 +1202,8 @@ void SecureContext::SetFreeListLength(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+// Currently, EnableTicketKeyCallback and TicketKeyCallback are only present for
+// the regression test in test/parallel/test-https-resume-after-renew.js.
```

Comment:
[suggestion] If i understand this case correctly add `, they are considered private until decided otherwise.`
Also maybe add `_` to the JS binding.